test/typings/tsconfig.5.5.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.4.7.json",
```

Comment:
Once we drop 4.7, we need to adjust the name of this base config. Let's see if it's specific to 4.7 in any way. Maybe it's better off as `tsconfig.base.json` instead.